packages/docs/src/App.js

```diff
@@ -38,6 +38,20 @@ const App = () => {
 
   document.querySelector('#favicon').href = `favicon-${theme}.png`
 
+  React.useEffect(() => {
+    const themeVal = window.localStorage.getItem("theme")
+    if(themeVal){
+      setThemeValue(themeVal)
+    }else{
+      setThemeValue('light')
+    }
+  },[])
+
+  const setThemeValue = theme => {
+    setTheme(theme)
+    window.localStorage.setItem("theme", theme);
+  }
+
```

Comment:
I would probably do it this a bit differently to simplify this:



```js
// pick up theme from localstorage (reading from window is safe because docs aren't server rendered)
const defaultTheme = window.localStorage.getItem("theme") || 'light'
const [theme, setTheme] = React.useState(defaultTheme)
```

and then

```js
// persist changes to theme in localstorage
React.useEffect(() => {    
  window.localStorage.setItem("theme", theme)
}, [theme])
```